mantidproject/mantid
# Fix pseudovoigt

**Description of work.**

Pseudo-Voigt function in Mantid does not have a detailed description on its math.  The current implemented Pseudo-Voigt equation does not have any literature or paper that can back it.  Especially the mixing parameter cannot not reflect the ratio between Gaussian and Lorentzian.

**To test:**
1. Proof read the documentation about the proposed formula of Pseudo-Voigt
2. Check whether all the unit tests make sense
3. You may use this to fit a Gaussian, a Lorentzian and a Pseudo-Voigt.



Fixes #24145. 



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
